www/_template/getting-started.md

```diff
@@ -3,43 +3,278 @@ layout: layouts/main.njk
+
+Now install Snowpack to your `dev dependencies` with this command:
 
-## Basic commands
+```
+npm install --save-dev snowpack
+```
+
+> 💡 Tip: Snowpack can also be installed globally via `npm install -g snowpack`. But, we recommend installing locally in every project via `--save-dev`/`--dev`. You can run the Snowpack CLI locally via package.json "scripts", npm's `npx snowpack`, or via `yarn snowpack`.
+
+## Snowpack's development server
+
+In this step we'll add some files so we can showcase Snowpack's development server, an instant development environment for unbundled development. The development server only builds a file when it's requested by the browser. That means that Snowpack can start up instantly (usually in **<50 ms**) and scale to infinitely large projects without slowing down. In contrast, it's common to see 30+ second dev startup times when building large apps with a traditional bundler.
+
+First create an `index.html` in your project with the following contents:
+
+```html
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta name="description" content="Starter Snowpack App" />
+    <title>Starter Snowpack App</title>
+  </head>
+  <body>
+    <h1>Welcome to Snowpack!</h1>
+  </body>
+</html>
+```
+
+Add the Snowpack development server to `package.json` under as the `start` script:
+
+```diff
+  "scripts": {
++   "start": "snowpack dev",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+
+```
 
-Here's a short list of what you can do with Snowpack:
+Run the following on the command line to start the Snowpack development server
+
+```
+npm run start
+```
+
+Snowpack displays the local host address and automatically opens the page in your default browser.
+
+IMAGE: GIF. Side by side of the terminal showing the dev server output. The dev server output displays the localhost address the project is running on. In a browser window you can see the running project on localhost, which is "Welcome to Snowpack" on a white background. An edit is made to `index.html` adding an exclamation point, the browser window shows the updated text as soon as the file is saved.
+
+Try changing the index.html and saving while the server is running, the site should refresh and show changes automatically.
+
+## Using ESM (ES Modules)
+
+You're ready to add some ES modules (ESM). JavaScript's native ES Module (ESM) syntax is the magic behind Snowpack's unbundled development. ESM lets you define explicit imports & exports that browsers and build tools can better understand and optimize for. If you're familiar with the `import` and `export` keywords in JavaScript, then you already know ESM!
+
+All modern browsers support ESM, so Snowpack is able to ship this code directly to the browser during development. This is what makes Snowpack's **unbundled development** workflow possible.
```

Comment:
RE the comment above about browser support: This is perfect, I'd say just remove that earlier blurb entirely.